lib/styles/colors.dart

```diff
@@ -0,0 +1,8 @@
+import 'dart:ui';
+
+class CocktailColors {
+  static const Color white = Color(0xFFFFFFFF);
+  static const Color background = Color(0xFFF5F5F5);
+  static const Color divider = Color(0xFF39E6D5);
```

Comment:
Is this divider color reused in more areas for different forms? If so, maybe we can think of a name that can make sense for the different things, or if not then it fine to leave like that for now.

Reply:
To be honest I didn't check that, but if I have to change name for one more reusable I will